### PR TITLE
Add mvn_test wrapper function

### PR DESCRIPTION
**Pull request by stdweird**
Add a hardended and convenient bash function to run unittests

### CODE REVIEW

File: build-scripts/src/main/perl/mvn_test.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#
```
Comment (jrha) [inline build-scripts/src/main/perl/mvn_test.sh]: Is this intended to be POSIX compliant `sh` or `bash`? It certainly looks like bash, a shebang wouldn't hurt.

File: build-scripts/src/main/perl/mvn_test.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+function mvn_pack()
+{
+    mvn_run "clean package -Dprove.args=-v $MVN_ARGS"
+}
+
+function _mvn_test_complete()
+{
+    local search
+    search="src/test/perl/${COMP_WORDS[COMP_CWORD]}"
+    COMPREPLY=( $(compgen -f $search | grep -E '\.t$' | sed 's#^src/test/perl/##;s/\.t$//' ) )
```
Comment (jrha) [inline build-scripts/src/main/perl/mvn_test.sh]: Should `$search` be double-quoted to avoid splitting and globbing?

**Comment by jrha** [comment]
@stdweird requires rebase

**Comment by jrha** [comment]
@stdweird *nudge*

**Comment by stdweird** [comment]
@wpoely86 ping

File: build-scripts/src/main/perl/mvn_test.sh (hunk 1 above)
Comment (jrha) [inline build-scripts/src/main/perl/mvn_test.sh]:
```suggestion
#!/bin/bash
#
```

File: build-scripts/src/main/perl/mvn_test.sh (hunk 2 above)
Comment (jrha) [reply build-scripts/src/main/perl/mvn_test.sh]:
```suggestion
    COMPREPLY=( $(compgen -f "$search" | grep -E '\.t$' | sed 's#^src/test/perl/##;s/\.t$//' ) )
```

**Comment by jrha** [comment]
@stdweird could you rebase and consider comments?

File: package-build-scripts/pom.xml [hunk 3]
```diff
@@ -3,12 +3,12 @@
   <groupId>org.quattor.client.libraries</groupId>
   <artifactId>perl-Test-Quattor</artifactId>
   <packaging>pom</packaging>
-  <version>16.4.1-SNAPSHOT</version>
+  <version>18.3.0-SNAPSHOT</version>
```
Comment (jrha) [inline package-build-scripts/pom.xml:6]:
```suggestion
  <version>21.12.1-SNAPSHOT</version>
```

File: package-build-scripts/pom.xml [hunk 4]
```diff
@@ -3,12 +3,12 @@
...
   <name>Test framework for Quattor</name>
   <parent>
     <groupId>org.quattor.maven</groupId>
     <artifactId>build-profile</artifactId>
-    <version>1.49</version>
+    <version>1.56</version>
```
Comment (jrha) [inline package-build-scripts/pom.xml:11]:
```suggestion
    <version>1.61</version>
```

**Review by jrha — APPROVED** [review]

**Pull request merged**